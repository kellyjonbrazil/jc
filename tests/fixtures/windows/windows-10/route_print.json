{
  "interface_list": [
    {
      "interface_index": 28,
      "mac_address": null,
      "description": "Tailscale Tunnel"
    },
    {
      "interface_index": 12,
      "mac_address": "00:1c:42:da:01:6a",
      "description": "Parallels VirtIO Ethernet Adapter"
    },
    {
      "interface_index": 1,
      "mac_address": null,
      "description": "Software Loopback Interface 1"
    }
  ],
  "ipv4_route_table": {
    "active_routes": [
      {
        "network_destination": "0.0.0.0",
        "netmask": "0.0.0.0",
        "gateway": "10.211.55.1",
        "interface": "10.211.55.3",
        "metric": "15"
      },
      {
        "network_destination": "10.0.0.0",
        "netmask": "255.0.0.0",
        "gateway": "192.168.22.1",
        "interface": "10.211.55.3",
        "metric": "16"
      },
      {
        "network_destination": "10.0.1.0",
        "netmask": "255.255.255.0",
        "gateway": "192.168.22.1",
        "interface": "10.211.55.3",
        "metric": "16"
      },
      {
        "network_destination": "10.0.3.0",
        "netmask": "255.255.255.0",
        "gateway": "192.168.22.1",
        "interface": "10.211.55.3",
        "metric": "16"
      },
      {
        "network_destination": "10.0.4.0",
        "netmask": "255.255.255.0",
        "gateway": "192.168.22.1",
        "interface": "10.211.55.3",
        "metric": "16"
      },
      {
        "network_destination": "10.211.55.0",
        "netmask": "255.255.255.0",
        "gateway": "On-link",
        "interface": "10.211.55.3",
        "metric": "271"
      },
      {
        "network_destination": "10.211.55.3",
        "netmask": "255.255.255.255",
        "gateway": "On-link",
        "interface": "10.211.55.3",
        "metric": "271"
      },
      {
        "network_destination": "10.211.55.255",
        "netmask": "255.255.255.255",
        "gateway": "On-link",
        "interface": "10.211.55.3",
        "metric": "271"
      },
      {
        "network_destination": "127.0.0.0",
        "netmask": "255.0.0.0",
        "gateway": "On-link",
        "interface": "127.0.0.1",
        "metric": "331"
      },
      {
        "network_destination": "127.0.0.1",
        "netmask": "255.255.255.255",
        "gateway": "On-link",
        "interface": "127.0.0.1",
        "metric": "331"
      },
      {
        "network_destination": "127.255.255.255",
        "netmask": "255.255.255.255",
        "gateway": "On-link",
        "interface": "127.0.0.1",
        "metric": "331"
      },
      {
        "network_destination": "169.254.0.0",
        "netmask": "255.255.0.0",
        "gateway": "On-link",
        "interface": "169.254.83.107",
        "metric": "261"
      },
      {
        "network_destination": "169.254.83.107",
        "netmask": "255.255.255.255",
        "gateway": "On-link",
        "interface": "169.254.83.107",
        "metric": "261"
      },
      {
        "network_destination": "169.254.255.255",
        "netmask": "255.255.255.255",
        "gateway": "On-link",
        "interface": "169.254.83.107",
        "metric": "261"
      },
      {
        "network_destination": "224.0.0.0",
        "netmask": "240.0.0.0",
        "gateway": "On-link",
        "interface": "127.0.0.1",
        "metric": "331"
      },
      {
        "network_destination": "224.0.0.0",
        "netmask": "240.0.0.0",
        "gateway": "On-link",
        "interface": "10.211.55.3",
        "metric": "271"
      },
      {
        "network_destination": "255.255.255.255",
        "netmask": "255.255.255.255",
        "gateway": "On-link",
        "interface": "127.0.0.1",
        "metric": "331"
      },
      {
        "network_destination": "255.255.255.255",
        "netmask": "255.255.255.255",
        "gateway": "On-link",
        "interface": "10.211.55.3",
        "metric": "271"
      }
    ],
    "persistent_routes": [
      {
        "network_address": "10.0.1.0",
        "netmask": "255.255.255.0",
        "gateway_address": "192.168.22.1",
        "metric": "1"
      },
      {
        "network_address": "10.0.3.0",
        "netmask": "255.255.255.0",
        "gateway_address": "192.168.22.1",
        "metric": "1"
      },
      {
        "network_address": "10.0.4.0",
        "netmask": "255.255.255.0",
        "gateway_address": "192.168.22.1",
        "metric": "1"
      },
      {
        "network_address": "10.0.0.0",
        "netmask": "255.0.0.0",
        "gateway_address": "192.168.22.1",
        "metric": "1"
      }
    ]
  },
  "ipv6_route_table": {
    "active_routes": [
      {
        "interface": 1,
        "metric": "331",
        "network_destination": "::1/128",
        "gateway": "On-link"
      },
      {
        "interface": 12,
        "metric": "271",
        "network_destination": "2001:db8::/64",
        "gateway": "fe80::1"
      },
      {
        "interface": 12,
        "metric": "271",
        "network_destination": "fdb2:2c26:f4e4::/64",
        "gateway": "On-link"
      },
      {
        "interface": 12,
        "metric": "271",
        "network_destination": "fdb2:2c26:f4e4:0:670f:fec7:75b7:eb83/128",
        "gateway": "On-link"
      },
      {
        "interface": 12,
        "metric": "271",
        "network_destination": "fdb2:2c26:f4e4:0:b4d5:8083:6ce6:37d/128",
        "gateway": "On-link"
      },
      {
        "interface": 12,
        "metric": "271",
        "network_destination": "fe80::/64",
        "gateway": "On-link"
      },
      {
        "interface": 12,
        "metric": "271",
        "network_destination": "fe80::4abd:fa70:3d36:83fd/128",
        "gateway": "On-link"
      },
      {
        "interface": 1,
        "metric": "331",
        "network_destination": "ff00::/8",
        "gateway": "On-link"
      },
      {
        "interface": 12,
        "metric": "271",
        "network_destination": "ff00::/8",
        "gateway": "On-link"
      }
    ],
    "persistent_routes": [
      {
        "interface": 0,
        "metric": "4294967295",
        "network_destination": "2001:db8::/64",
        "gateway": "fe80::1"
      }
    ]
  }
}
