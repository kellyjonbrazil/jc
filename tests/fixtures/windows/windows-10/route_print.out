===========================================================================
Interface List
 28...........................Tailscale Tunnel
 12...00 1c 42 da 01 6a ......Parallels VirtIO Ethernet Adapter
  1...........................Software Loopback Interface 1
===========================================================================

IPv4 Route Table
===========================================================================
Active Routes:
Network Destination        Netmask          Gateway       Interface  Metric
          0.0.0.0          0.0.0.0      10.211.55.1      10.211.55.3     15
         10.0.0.0        255.0.0.0     192.168.22.1      10.211.55.3     16
         10.0.1.0    255.255.255.0     192.168.22.1      10.211.55.3     16
         10.0.3.0    255.255.255.0     192.168.22.1      10.211.55.3     16
         10.0.4.0    255.255.255.0     192.168.22.1      10.211.55.3     16
      10.211.55.0    255.255.255.0         On-link       10.211.55.3    271
      10.211.55.3  255.255.255.255         On-link       10.211.55.3    271
    10.211.55.255  255.255.255.255         On-link       10.211.55.3    271
        127.0.0.0        255.0.0.0         On-link         127.0.0.1    331
        127.0.0.1  255.255.255.255         On-link         127.0.0.1    331
  127.255.255.255  255.255.255.255         On-link         127.0.0.1    331
      169.254.0.0      255.255.0.0         On-link    169.254.83.107    261
   169.254.83.107  255.255.255.255         On-link    169.254.83.107    261
  169.254.255.255  255.255.255.255         On-link    169.254.83.107    261
        224.0.0.0        240.0.0.0         On-link         127.0.0.1    331
        224.0.0.0        240.0.0.0         On-link       10.211.55.3    271
  255.255.255.255  255.255.255.255         On-link         127.0.0.1    331
  255.255.255.255  255.255.255.255         On-link       10.211.55.3    271
===========================================================================
Persistent Routes:
  Network Address          Netmask  Gateway Address  Metric
         10.0.1.0    255.255.255.0     192.168.22.1       1
         10.0.3.0    255.255.255.0     192.168.22.1       1
         10.0.4.0    255.255.255.0     192.168.22.1       1
         10.0.0.0        255.0.0.0     192.168.22.1       1
===========================================================================

IPv6 Route Table
===========================================================================
Active Routes:
 If Metric Network Destination      Gateway
  1    331 ::1/128                  On-link
 12    271 2001:db8::/64            fe80::1
 12    271 fdb2:2c26:f4e4::/64      On-link
 12    271 fdb2:2c26:f4e4:0:670f:fec7:75b7:eb83/128
                                    On-link
 12    271 fdb2:2c26:f4e4:0:b4d5:8083:6ce6:37d/128
                                    On-link
 12    271 fe80::/64                On-link
 12    271 fe80::4abd:fa70:3d36:83fd/128
                                    On-link
  1    331 ff00::/8                 On-link
 12    271 ff00::/8                 On-link
===========================================================================
Persistent Routes:
 If Metric Network Destination      Gateway
  0 4294967295 2001:db8::/64            fe80::1
===========================================================================
